cpp/src/bitmask/null_mask.cu

```diff
@@ -275,6 +291,7 @@ rmm::device_buffer create_null_mask(size_type size,
                                     rmm::cuda_stream_view stream,
                                     rmm::device_async_resource_ref mr)
 {
+  CUDF_FUNC_RANGE();
```

Comment:
Use `CUDF_FUNC_RANGE()` in the public API's definition before calling the detail API instead of within detail APIs possibly leading to nested `CUDF_FUNC_RANGE()`s as well as consistency